src/pages/kwonojae/Main/components/Feed.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faEllipsisH } from '@fortawesome/free-solid-svg-icons';
+import {
+  faPaperPlane,
+  faHeart,
+  faComment,
+  faBookmark,
+} from '@fortawesome/free-regular-svg-icons';
+import Writecomment from './Writecomment';
+import './Feed.scss';
+
+class Feed extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      feedList: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/kwonojae/feeddata.json', {
+      method: 'GET',
+    })
+      .then(response => response.json())
+      .then(feedData => {
+        this.setState({ feedList: feedData });
+      });
+  }
+
+  render() {
+    const { feedList } = this.state;
+
+    return (
+      <section className="feedOJ">
+        {feedList.map(feedList => {
+          return (
+            <article key={feedList.id}>
```

Comment:
```suggestion
      <section className="feedOJ">
        {feedList.map(feed => {
          return (
            <article key={feed.id}>
```
feedList에 map을 적용하면, 인자는 feed로 쓰는게 더욱 좋지 않을까 싶습니다!

Reply:
그러네요 ㅎ 수정했습니다~